lib/taskana-core/src/test/java/acceptance/task/GetTaskCommentAccTest.java

```diff
@@ -89,8 +90,11 @@ void testGetNonExistingTaskCommentShouldFail() {
 
     TaskService taskService = taskanaEngine.getTaskService();
 
-    assertThatThrownBy(() -> taskService.getTaskComment("Definately Non Existing Task Comment Id"))
-        .isInstanceOf(TaskCommentNotFoundException.class);
+    ThrowingCallable lambda =
+        () ->
+            taskService.getTaskComment(
+                "Invalid Task Id", "Definately Non Existing Task Comment Id");
+    assertThatThrownBy(lambda).isInstanceOf(TaskCommentNotFoundException.class);
```

Comment:
Should we add 2 more test cases with a valid task id + invalid comment id & invalid / wrong taskid + valid comment id?

Reply:
I added a test for a valid taskId+invalid taskCommentId, good idea :). A test for wrong taskId + valid taskCommentId doesn't make much sense to me, since this combination basically can't exist. A task comment only exists, when a task also exists. How can there be a valid taskCommentId (and therefore a existing taskComment) when the task doesn't exist?